packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -523,9 +522,9 @@ class ViewportPositioner extends Foundation<
     private getVerticalPositionAvailableHeight = (
         positionOption: ViewportPositionerVerticalPositionLabel
     ): number => {
-        const spaceAbove: number = this.anchorTop - this.viewportRect.top;
+        const spaceAbove: number = this.anchorTop;
         const spaceBelow: number =
-            this.viewportRect.bottom - (this.anchorTop + this.anchorHeight);
+            this.viewportRect.height - spaceAbove - this.anchorHeight;
```

Comment:
I'd like it if we could extrapolate this logic in a way that we can test it

Reply:
I'll add some function tests